mwdb/model/user.py

```diff
@@ -210,12 +210,11 @@ def verify_session_token(token) -> Optional[Tuple["User", Optional[str]]]:
 
     @staticmethod
     def verify_set_password_token(token) -> Optional["User"]:
-        result = User._verify_token(
+        return User._verify_token(
             token,
             ["password_ver"],
             scope=AuthScope.set_password,
         )
-        return None if result is None else result[0]
```

Comment:
This complication is an inevitable consequence of changing the initial return type. Reusing the `_verify_token()` is now globally less convenient.